sharktank/sharktank/utils/iree.py

```diff
@@ -753,3 +759,251 @@ def run_model_with_iree_run_module(
     input_args = [f"--input={arg.strip()}" for arg in input_args]
     cmd += input_args
     subprocess.check_call(cmd, **subprocess_run_kwargs)
+
+
+def _as_tuple(x):
+    if isinstance(x, tuple):
+        return x
+    if isinstance(x, list):
+        return tuple(x)
+    return (x,)
+
+
+def export_torch_module_to_mlir(
```

Comment:
The place of this function is more appropriate in `utils/export.py`. It seems that you are trying to make another overload of the underlying of IREE Turbine's [export](https://github.com/iree-org/iree-turbine/blob/c80577fdc61a1565cd906205f82c8c79c62f5aea/iree/turbine/aot/exporter.py#L171) that can export to a file instead of returning an `ExportOutput` object. I suggest that you use the same arguments as the underlying export function with the addition of `mlir_path`. The we don't need deal with a slightly different naming of arguments like `target_fn` instead of `function_name`.